config/samples/dataplane_v1beta1_openstackdataplane.yaml

```diff
@@ -35,17 +43,71 @@ spec:
         ansiblePort: 22
         ansibleSSHPrivateKeySecret: dataplane-ansible-ssh-private-key-secret
         ansibleVars: |
+          service_net_map:
+            nova_api_network: internal_api
+            nova_libvirt_network: internal_api
+
+          # edpm_network_config
+          # Default nic config template for a EDPM compute node
+          # These vars are edpm_network_config role vars
           edpm_network_config_template: templates/net_config_bridge.j2
```

Comment:
this should be the default value from install_yamls shouldn't it?
`templates/single_nic_vlans/single_nic_vlans.j2`

Reply:
done!